app/Repositories/MessageRepository.php

```diff
@@ -120,6 +120,12 @@ public function updateMessagesForContest($contest, $messages)
                 $data['type'] = $type;
                 $data['key'] = $key;
 
+                if ($data['type'] === 'leaderboard') {
+                    if (! array_key_exists('show_images', $data)) {
+                        $data['show_images'] = 0;
```

Comment:
https://laravel.com/docs/5.2/eloquent-mutators#accessors-and-mutators from the docs and checking now for in Gladiator.